atest/testdata/libdoc/TocLibraryInitRobot.py

```diff
@@ -0,0 +1,23 @@
+class TocLibraryInitRobot(object):
+    """Simple library
+
+    %TABLE_OF_CONTENTS%
```

Comment:
I think I proposes `%TABLE_OF_CONTENTS%` instead of `%TOC%` but now that I see it live it looks a bit ugly. What do you think, should we use `%TOC%` anyway?

Reply:
I am happy with both ways, but I agree that `%TOC%` looks better. Will change, not a big thing.